ember-primitives/src/components/accordion/accordion.gts

```diff
@@ -0,0 +1,150 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { hash } from '@ember/helper';
+
+// temp
+//  https://github.com/tracked-tools/tracked-toolbox/issues/38
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { localCopy } from 'tracked-toolbox';
+
+import AccordionItem from './item';
+
+import type { WithBoundArgs } from '@glint/template';
+
+type AccordionSingleArgs = {
+  /**
+   * The type of accordion. If `single`, only one item can be selected at a time. If `multiple`, multiple items can be selected at a time.
+   */
+  type: 'single',
+  /**
+   * Whether the accordion is disabled.
```

Comment:
what happens then disabled is true? could that information be added to the text here?

Reply:
Setting the accordion to disabled causes all items to no longer be expandable/collapsible.

Added this to the JSDoc!